aten/src/ATen/test/quantized_test.cpp

```diff
@@ -45,6 +46,24 @@ TEST(TestQTensor, QuantDequantAPIs) {
   for (auto i = 0; i < num_elements; ++i) {
     ASSERT_EQ(r_data[i], rqr_data[i]);
   }
+  for (auto i = 0; i < num_elements; ++i) {
+    ASSERT_EQ(r_data[i],
+              dequantize_val(qr.q_scale(), qr.q_zero_point(), qr_data[i]));
+  }
+
+  // Check for correct requantization
+  double new_scale = 2.0;
+  int64_t new_zero_point = 1;
+  Tensor reqr = at::quantize_linear(r, new_scale, new_zero_point, kQInt8);
+  auto reqr_data = reqr.data<qint8>();
+  for (auto i = 0; i < num_elements; ++i) {
+    reqr_data[i].val_ = requantize_val<quint8, qint8>(scale, zero_point,
```

Comment:
can you fix the formatting?

Reply:
Can you point to the actual formatting issue, please?

Reply:
looks like the arguments are expanded into multiple lines and doesn't align. not sure whether this is just github display it this way or not. just make sure it looks ok in your editor

Reply:
Resolved in person